packages/flutter/lib/src/widgets/modal_barrier.dart

```diff
@@ -12,6 +12,19 @@ import 'navigator.dart';
 import 'transitions.dart';
 
 /// A widget that prevents the user from interacting with widgets behind itself.
+///
+/// The modal barrier is the scrim that is rendered behind each route, which
```

Comment:
Is 'scrim' a typo or just a term I should learn?

Reply:
It's a term of art in theatre. Not sure what better term to use here.